vlib/v/vet/tests/module_file_test.out

```diff
@@ -0,0 +1,5 @@
+vlib/v/vet/tests/module_file_test.vv:7: Function documentation seems to be missing for "pub fn foo() string".
+vlib/v/vet/tests/module_file_test.vv:13: A function name is missing from the documentation of "pub fn bar() string".
+vlib/v/vet/tests/module_file_test.vv:35: Function documentation seems to be missing for "pub fn (f Foo) foo() string".
+vlib/v/vet/tests/module_file_test.vv:46: A function name is missing from the documentation of "pub fn (f Foo) fooo() string".
+NB: You can run `v fmt -w file.v` to fix these automatically
```

Comment:
I am pretty sure that this shouldn't be here? 🤔

Reply:
Nope, but the code currently doesn't allow me to remove it - since that will break all the other tests.
I like it when programs are being helpful - but it doesn't cover all cases :shrug:

Reply:
I think parse_vet_file should be moved out of parser.v, to cmd/tools/vvet.v or to vlib/v/vet.v, to facilitate easier maintenance of `vet`, and to make the parser slightly smaller.

Reply:
Perhaps moving cmd/tools/vvet.v to cmd/tools/vvet/vvet.v and moving that function there too (similar to how vdoc was refactored) ?
`vlib/v/vet` does not appear to have anything except a test, that can also be moved inside the new cmd/tools/vved .

Reply:
Agreed, they look out of place in the parser. I'll see what I can do 🙂

Reply:
I wasn't able to move parse_vet_file out as it gave me problems with fields not being pub. Also it's made so the scanner and parser collects errors in vet mode and vvet can then do extra validating etc.